### PR TITLE
markdig fixes - part 2

### PR DESCRIPTION
**Pull request by mairaw**
/cc @scottaddie 

Hide whitespace changes

### CODE REVIEW

File: docs/framework/unmanaged-api/debugging/icordebugmodule3-interface.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ interface ICorDebugModule3 : IUnknown
   
  **Library:** CorGuids.lib  
   
- **.NET Framework Versions:**4.5, 4, 3.5 SP1  
+ **.NET Framework Versions:** 4.5, 4, 3.5 SP1
```
Comment (scottaddie) [inline docs/framework/unmanaged-api/debugging/icordebugmodule3-interface.md]: While you're in here, should "4" be "4.0"?

File: docs/framework/debug-trace-profile/enhancing-debugging-with-the-debugger-display-attributes.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ author: "mairaw"
...
+Debugger display attributes allow the developer of the type, who specifies and best understands the runtime behavior of that type, to also specify what that type will look like when it is displayed in a debugger. In addition, debugger display attributes that provide a `Target` property can be applied at the assembly level by users without knowledge of the source code. The <xref:System.Diagnostics.DebuggerDisplayAttribute> attribute controls how a type or member is displayed in the debugger variable windows. The <xref:System.Diagnostics.DebuggerBrowsableAttribute> attribute determines if and how a field or property is displayed in the debugger variable windows. The <xref:System.Diagnostics.DebuggerTypeProxyAttribute> attribute specifies a substitute type, or a proxy, for a type and changes the way the type is displayed in debugger windows. When you view a variable that has a proxy, or substitute type, the proxy stands in for the original type in the debugger display window. The debugger variable windowdisplays only the public members of the proxy type. Private members are not displayed.
```
Comment (rpetrusha) [inline docs/framework/debug-trace-profile/enhancing-debugging-with-the-debugger-display-attributes.md]: nit unrelated to your PR: windowdisplays --> window displays

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. I left one nit unrelated to your PR. You can merge when you're ready.

File: docs/framework/unmanaged-api/debugging/icordebugmodule3-interface.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/unmanaged-api/debugging/icordebugmodule3-interface.md]: 4 is kind of the official name of that release. I'd have to look into other places we use 4 and change it to 4.0 at once.

**Pull request merged**